src/main/scala/IOService/Http4sTestServer.scala

```diff
@@ -0,0 +1,109 @@
+package IOService
+import cats._
+import cats.effect._
+import cats.implicits._
```

Comment:
Not used to seeing `cats.implicit._` anymore. Is this deliberate?

Reply:
Not deliberate.  I just grabbed some "suggested" imports.  Probably I'm still going off old docs. :(

Reply:
Most cases of `cats.implicits._` can be replaced by `cats.syntax.all._` since Cats 2.2.  `implicits` is `syntax.all` plus `instances.all`, but most of the instances on `instances.all` moved to the type class companions and are already in scope.  There's nothing harmful about this import, but it's last season's fashion.

Reply:
That's right.  Ty, sir!